frontend/src/views/v-authorship.vue

```diff
@@ -119,21 +127,31 @@
                 font-awesome-icon.button(icon="user-edit")
                 span.tooltip-text Click to view the blame view of file
           span.fileTypeLabel(
-            v-if='!file.isBinary',
+            v-if='!file.isBinary && !file.isIgnored',
             v-bind:style="{\
               'background-color': fileTypeColors[file.fileType],\
               'color': getFontColor(fileTypeColors[file.fileType])\
               }"
           ) {{ file.fileType }}&nbsp;{{ file.lineCount }}
             |&nbsp;({{ file.lineCount - file.blankLineCount }})
           span.fileTypeLabel.binary(v-if='file.isBinary') binary&nbsp;
-        pre.hljs.file-content(v-if="file.wasCodeLoaded && !file.isBinary", v-show="file.active")
+          span.fileTypeLabel(
+            v-if='file.isIgnored',
+            v-bind:style="{\
+              'background-color': '#000000',\
+              'color': '#ffffff'\
+              }"
+          ) ignored ({{ file.lineCount }})&nbsp;
+        pre.hljs.file-content(v-if="file.wasCodeLoaded && !file.isBinary && !file.isIgnored", v-show="file.active")
```

Comment:
Would [v-else-if](https://v2.vuejs.org/v2/guide/conditional.html) be better?

Reply:
Fixed